configs/torch-2.8/torch_variant_config.toml

```diff
@@ -64,6 +86,28 @@ deps_add_list    = [
     'nvidia-nvtx==12.9.79; platform_system == "Linux" and platform_machine == "x86_64" and "nvidia :: cuda_version_lower_bound :: 12.0" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.8" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.9" in variant_properties',
     'nvidia-nvjitlink==12.9.86; platform_system == "Linux" and platform_machine == "x86_64" and "nvidia :: cuda_version_lower_bound :: 12.0" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.8" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.9" in variant_properties',
     'nvidia-cufile==1.14.1.1; platform_system == "Linux" and platform_machine == "x86_64" and "nvidia :: cuda_version_lower_bound :: 12.0" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.8" not in variant_properties and "nvidia :: cuda_version_lower_bound :: 12.9" in variant_properties',
+
+    # XPU
+    'dpcpp-cpp-rt==2025.1.1; platform_system == "Linux" and "intel" in variant_namespaces',
+    'impi-rt==2021.15.0; platform_system == "Linux" and platform_machine == "x86_64" and "intel" in variant_namespaces',
```

Comment:
If anyone curious, few lines have additional `platform_machine == "x86_64"` because that's how original xpu non-variant wheel has specified these dependencies. See here: https://github.com/wheelnext/variant-repack/pull/3#issuecomment-3198830400. I basically copied xpu specific deps here and added 2 things: namespace protection (`"intel" in variant_namespaces'`) and affinity to Linux (`platform_system == "Linux"`).